### PR TITLE
deploy buildbot.buildbot.net with 1.x branch and python3

### LINKED ISSUE #212: Migrate metabb from nine.buildbot.net to buildbot.buildbot.net
Migrate the meta-buildbot instance from nine.buildbot.net back to buildbot.buildbot.net. The nine subdomain was for testing nine but that is now the primary and only instance. buildbot.buildbot.net redirects to nine.buildbot.net and I think that should be flipped. It's a 302 redirect so it's not permanent and safe to switch.

- [x] Get new certificates.
- [ ] Add nine -> buildbot redirect.
- [x] Update GitHub hook.

### LINKED ISSUE #216: nine.buildbot.net needs to be converted to running Python 3
http://nine.buildbot.net/#/about

is running Python 2.7.14

It needs to be converted to run Python 3

### PR DESCRIPTION
**Pull request by tardyp**
fixes #216 
fixes #212 

Added a method to avoid redploying the master if it is busy

### CODE REVIEW

File: roles/base/tasks/ansible-pull.yml [hunk 1]
```diff
@@ -46,7 +46,7 @@
   git:
     repo: "{{ ansible_git_repository }}"
     dest: "/home/{{ service_account }}/repo"
-    version: master
```
Comment (seankelly) [inline roles/base/tasks/ansible-pull.yml]: Need to switch this back to `master` for the merge.

File: roles/virtualenv3/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+---
+- name: make sure we have python3.6
+  pkgng:
+    name: "{{item}}"
+    state: present
+  with_items: 
+    - python36
+    - py27-virtualenv
```
Comment (seankelly) [inline roles/virtualenv3/tasks/main.yml]: `py27-virtualenv`? Should this be `py36-virtualenv`?

File: roles/virtualenv3/tasks/main.yml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  environment: "{{ proxy_env }}"
+
+- name: create the virtual environment
+  become: yes
+  become_user: "{{ venv_user }}"
+  pip:
+    name: "{{ item }}"
+    editable: false
+    virtualenv: "{{ venv_home_dir }}/{{ venv_name }}"
+    virtualenv_python: "python3.6"
```
Comment (seankelly) [inline roles/virtualenv3/tasks/main.yml]: There's also a `virtualenv_command` option where you can pick `virtualenv-3.6`. Doing that automatically selects python 3.6 for the virtualenv.

File: roles/base/tasks/ansible-pull.yml (hunk 1 above)
Comment (tardyp) [reply roles/base/tasks/ansible-pull.yml]: yes, the task is still WIP, I need to squash the hacks..

File: roles/virtualenv3/tasks/main.yml (hunk 2 above)
Comment (tardyp) [reply roles/virtualenv3/tasks/main.yml]: virtualenv is part of python2. with python3 that is pyenv, and I am not sure how well it is supported by ansible.

File: roles/virtualenv3/tasks/main.yml (hunk 3 above)
Comment (tardyp) [reply roles/virtualenv3/tasks/main.yml]: yeah, I did not try this path yet

File: roles/virtualenv3/tasks/main.yml (hunk 2 above)
Comment (seankelly) [reply roles/virtualenv3/tasks/main.yml]: The `virtualenv_command` comments mention `pyvenv`. It appears you can have `virtualenv_command: "-m venv"` too.

Comment (tardyp) [reply roles/virtualenv3/tasks/main.yml]: I used that:
https://stackoverflow.com/questions/42579168/ansible-create-a-virtualenv-using-the-venv-module

**Comment by tardyp** [comment]
@seankelly I think this is now ready. buildbot.buildbot.net is in prod and up and running!

**Comment by seankelly** [comment]
Great!

**Pull request merged**